atomic_reactor/plugins/pre_download_remote_source.py

```diff
@@ -63,17 +63,16 @@ def generate_cachito_env_file(self):
         cachito request.
         """
 
-        if self.buildargs:
```

Comment:
Could the unit test be parametrized (with buildargs, without buildargs)?

Reply:
Unit test added. PTAL